Microsoft.Toolkit.Uwp.UI.Controls/Expander/Expander.cs

```diff
@@ -148,7 +149,10 @@ private void OnExpandDirectionChanged(bool useTransitions = true)
 
         private void OnContentOverlayChanged()
         {
-            UpdateOverlayAlignment();
+            if (GetTemplateChild(ContentOverlayPart) is Grid contentOverlay)
+            {
+                contentOverlay.Visibility = ContentOverlay != null ? Visibility.Visible : Visibility.Collapsed;
```

Comment:
We can use the EmptyObjectToObjectConverter to create a 'NullToVisibilityConverter' and do this all in the template.  Then we can get rid of the ContentOverlay part entirely.

Reply:
https://docs.microsoft.com/en-us/windows/communitytoolkit/helpers/converters#emptyobjecttoobjectconverter-examples

Reply:
Good idea, done. Please review again